tools/testinfra/runner/src/buck_test.rs

```diff
@@ -25,8 +25,11 @@ pub struct Test {
     /// The command used to execute the test.
     pub command: Command,
 
-    /// User-facing identifier for this specific test.
-    pub name: String,
+    /// Fully qualified buck test target.
+    pub target: String,
+
+    /// Unit test inside target.
+    pub unit: Option<String>,
```

Comment:
Will this ever be `None`? If it's `None` in a case like `buck_sh_test`, it would be better to just default to `main` or something, so that you can assume a unit test always has a target and a name within that target, that should make some other code simpler